RELEASE_NOTES.md

```diff
@@ -1,5 +1,214 @@
 # Release Notes
 
+## 4.8.0
+### Features
+- A new command, `podman farm build`, has been added, which sends out builds to nodes defined in the farm, builds the images on the farm nodes, and pulls them back to the local machine to create a manifest list.
```

Comment:
So what is the status on that actually, do we really support it? There are no docs for farm build and the command farm is hidden! My last knowledge is that it needs to be reworked: https://github.com/containers/podman/issues/20505#issuecomment-1783013915
cc @umohnani8

Reply:
if this is the case, it should not be a feature.

Reply:
It is considered Tech Preview for RHEL 8.10/9.4.  This means it should work, but if you run into a problem with it, we won't guarantee a fix under the standard RHEL turn-around requirements.  If we document it, I would suggest Tech Preview for 4.8 on other distros.   I don't think I'd pull all docs for it and have it available but not acknowledged.

Reply:
Let's have this discussion at Scrum tomorrow

Reply:
Apologies on missing this yesterday. Let's not add this to the release notes yet as the rework will be significantly different that the existing workflow. The command is also still hidden and would prefer to keep it that way until everything is working as expected.